test/extended/dr/static_pod_rollouts.go

```diff
@@ -0,0 +1,145 @@
+		etcdPodsCount := getPodCount(oc, etcdNamespace, "app=etcd")
+		o.Expect(etcdPodsCount).To(o.BeNumerically("==", 3), "three etcd instances should be available in the initial state of the cluster")
+
+		// select a node to remove the etcd instance from
+		masterNodes := masterNodes(oc)
+		etcdTargetNode := masterNodes[0]
+
+		g.GinkgoT().Log("debugging into the node to remove the etcd static pod manifest")
+		err = oc.AsAdmin().Run("debug").Args("node/"+etcdTargetNode.Name, "--", "chroot", "/host", "/bin/bash", "-c", "mkdir /var/lib/etcd-backup && mv /etc/kubernetes/manifests/etcd-pod.yaml /var/lib/etcd-backup").Execute()
+		err = errors.Wrap(err, "failed to remove the etcd static pod from the node")
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		// allow some time for the kubelet to act after removing the manifest
+		time.Sleep(60 * time.Second)
+
+		g.GinkgoT().Log("ensuring the successful removal of the etcd instance")
+		etcdPodsCount = getPodCount(oc, etcdNamespace, "app=etcd")
+		o.Expect(etcdPodsCount).To(o.BeNumerically("==", 2), "only two etcd instances should be available in the cluster after the removal of one instance")
+
+		g.GinkgoT().Log("ensuring that the removal of the etcd instance marks the corresponding etcd guard as not ready")
+		etcdGuardPod, err := oc.AdminKubeClient().CoreV1().Pods(etcdNamespace).List(ctx, metav1.ListOptions{LabelSelector: "app=guard", FieldSelector: "spec.nodeName=" + etcdTargetNode.Name})
+		o.Expect(err).ToNot(o.HaveOccurred())
+		o.Expect(len(etcdGuardPod.Items)).To(o.BeNumerically("==", 1), "only the correct etcd guard should be retrieved from the node")
+		isEtcdGuardReady := isPodReady(etcdGuardPod.Items[0])
+		o.Expect(isEtcdGuardReady).To(o.BeFalse(), "expected the etcd guard to be marked as not ready after the removal of respective etcd instance")
+
+		// retrieve the operator status
+		etcdCluster, err := oc.AdminOperatorClient().OperatorV1().Etcds().Get(ctx, "cluster", metav1.GetOptions{})
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		g.GinkgoT().Log("ensuring the EtcdCertSignerController has Degraded")
+		status := findCondition(etcdCluster.Status.Conditions, "EtcdCertSignerControllerDegraded")
+		o.Expect(string(status)).To(o.Equal(string(metav1.ConditionTrue)), "expected the EtcdCertSignerController to be degraded due to insufficient quorum")
+
+		g.GinkgoT().Log("ensuring the EtcdEndPointsController has Degraded")
+		status = findCondition(etcdCluster.Status.Conditions, "EtcdEndpointsDegraded")
+		o.Expect(string(status)).To(o.Equal(string(metav1.ConditionTrue)), "expected the EtcdEndPointsController to be degraded due to insufficient quorum")
+
+		// g.GinkgoT().Log("ensuring the TargetConfigController has Degraded")
+		// status = findCondition(etcdCluster.Status.Conditions, "TargetConfigControllerDegraded")
+		// o.Expect(string(status)).To(o.Equal(string(metav1.ConditionTrue)), "expected the TargetConfigController to be degraded due to insufficient quorum")
```

Comment:
it certainly complains about it in the logs:

```
[etcd-operator-5b688b595d-gpftb] E0731 09:37:43.094688       1 base_controller.go:268] TargetConfigController reconciliation failed: TargetConfigController can't evaluate whether quorum is safe: etcd cluster has quorum of 2 and 2 healthy members which is not fault tolerant: [{Member:ID:10672555535946225151 name:"ci-ln-3r1lwyt-72292-dnkwv-master-2" peerURLs:"https://10.0.0.4:2380" clientURLs:"https://10.0.0.4:2379"  Healthy:true Took:2.99623ms Error:<nil>} {Member:ID:3303868763609047786 name:"ci-ln-3r1lwyt-72292-dnkwv-master-1" peerURLs:"https://10.0.0.5:2380" clientURLs:"https://10.0.0.5:2379"  Healthy:true Took:3.55111ms Error:<nil>} {Member:ID:17377880278057170250 name:"ci-ln-3r1lwyt-72292-dnkwv-master-0" peerURLs:"https://10.0.0.3:2380" clientURLs:"https://10.0.0.3:2379"  Healthy:false Took: Error:create client failure: failed to make etcd client for endpoints [https://10.0.0.3:2379]: context deadline exceeded}]
```

I think we need to unit test this :) might be a bug